internal/pkg/pct/install.go

```diff
@@ -92,6 +96,53 @@ func (p *PctInstaller) Install(templatePkg string, targetDir string, force bool)
 	return namespacedPath, nil
 }
 
+func (p *PctInstaller) InstallClone(gitUri string, targetDir string, tempDir string, force bool) (string, error) {
+	// Validate git URI
+	_, err := url.ParseRequestURI(gitUri)
+	if err != nil {
+		return "", fmt.Errorf("Could not parse template uri %s: %v", gitUri, err)
+	}
+
+	// Clone git repository to temp folder
+	folderPath, err := p.cloneTemplate(gitUri, tempDir)
+	if err != nil {
+		return "", fmt.Errorf("Could not clone git repository: %v", err)
+	}
+
+	// Remove .git folder from cloned repository
+	err = p.AFS.RemoveAll(filepath.Join(folderPath, ".git"))
+	if err != nil {
+		return "", fmt.Errorf("Failed to remove '.git' directory")
+	}
+
+	// Read config to determine template properties
+	info, err := p.readConfig(filepath.Join(folderPath, "pct-config.yml"))
+	if err != nil {
+		return "", fmt.Errorf("Invalid config: %v", err.Error())
+	}
+
+	// Create namespaced directory and move contents of temp folder to it
+	namespacedPath, err := p.setupTemplateNamespace(targetDir, info, folderPath, force)
+	if err != nil {
+		return "", fmt.Errorf("Unable to install in namespace: %v", err.Error())
+	}
+
+	return namespacedPath, nil
+}
+
+func (p *PctInstaller) cloneTemplate(gitUri string, tempDir string) (string, error) {
+	// TODO: Sanitize command args
+	//clonePath := filepath.Clean(filepath.Join(tempDir, "temp"))
```

Comment:
Would remove these lines as the TODO action is no longer required and the `filepath.Clean` is not used